requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+attrs==19.1.0
```

Comment:
Needed packages for tests, safety and flake8 (for tox but also for developers to test locally or in VENV).